.storybook/themes/commerce.scss

```diff
@@ -168,13 +168,14 @@
 	$alias: (
 		"primary-color": var(--global-neutral-10),
 		"primary-font-family": "Inter",
+		"layout-content-max-width": 1216px,
```

Comment:
Remove the units here.  This is going to break the responsive alias in category-carousel... see next comment as well.

Reply:
```suggestion
		"layout-content-max-width": 1216,
```

Reply:
@nschubach : @JackHowa submitted a PR to this branch and we've got it fixed up. Thanks!